.github/services/swift/swift/action.yml

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: swift
+description: "Behavior test for OpenStack Swift"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Swift service
+      shell: bash
+      working-directory: fixtures/swift
+      run: |
+        docker compose -f docker-compose-swift.yml up -d --wait
+
+    - name: Create test token and setup test container
+      shell: bash
+      run: |
+        token=$(curl -i -H 'X-Storage-User: test:tester' -H 'X-Storage-Pass: testing' http://127.0.0.1:8080/auth/v1.0 | grep X-Auth-Token | head -n1 | awk '{print $2}')
+        echo "OPENDAL_SWIFT_TOKEN=$token" >> $GITHUB_ENV
+        curl --location --request PUT 'http://127.0.0.1:8080/v1/AUTH_test/testing' --header "X-Auth-Token: $OPENDAL_SWIFT_TOKEN"
```

Comment:
I'm guessing token here is empty.

Reply:
I originally thought the same way, but after adding `eprintln!("X-Auth-Token: {}", &self.token);`, it printed out the correct value, as shown below:
![image](https://github.com/apache/incubator-opendal/assets/10728152/c5364080-4720-4cd2-a276-1a12805dbc04)

Besides, if the value is empty, it shouldn't raise a `failed to parse header value` error.

Reply:
Will this token contains a new line?

Reply:
Confirmed it's a newline problem, I triaged it by adding `&self.token.trim_end().to_string()`, and it works. However, it took me a very long time to figure out how to make it work without changing the Rust code. To sum up, only `tr -d '[:space:]'` works, but not `tr -d '\n'`.